README.md

```diff
@@ -27,30 +27,31 @@ It's mighty because it:
 
 It's easy to get started.
 
-First, decide how you want to use stylelint:
+First, decide if you want to use stylelint:
```

Comment:
I think the user has _already_ decided to use stylelint if they are at this point...

The following 5 bullet items are "how" to use stylelint, they are not sentences, statements, or quotes to aid a user in deciding "if" they should use stylelint.

I suggest reverting this change back to `how`

Reply:
Yep, it's a odd one. I was following [this guidance](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps):

> - you always use a lead-in line
> - the bullets make sense running on from the lead-in line

"First, decide if you want to use stylelint on the CLI"
"First, decide if you want to use stylelint in your build tool"

Whereas "how" ends up meaning something else "First, decide how you want to use stylelint on the CLI". 

There's probably a better way of writing the lead-in line to remove the oddity you mentioned, but I can't think of one right now :) So, I'll revert the change.

Reply:
Ha, I guess so, language is hard and I only speak one!